libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -348,6 +348,13 @@ class YulUtilFunctions
 	/// @returns helper function that tries to decode the error message from a "structured" revert() instruction.
 	std::string tryDecodeErrorMessageFunction();
 
+
+	/// Returns a function name that wraps around returndatacopy().
+	///
+	/// If returndatacopy() is not supported by the underlying target, a zeroPointer will be
+	/// returned instead.
+	std::string returndatacopyFunction();
```

Comment:
maybe "extractReturndataFunction"? Or "copyReturndataFunction"? If we keep the name like this, it sounds like it just does 'returndatacopy'